### DISCUSSION TITLE
Big Sur

### DISCUSSION THREAD

**Discussion by tahaklkt-osxinfo** [chris1111/Legacy-Video-patch#46] [Q&A]
Hello mate. This is my last question. These kexts are works on Big Sur /S/L/E ? I can enable write access on root disk. Thanks!

**Comment by chris1111** [comment]
No I don't think

**Reply by tahaklkt-osxinfo** [reply]
OK thanks.

**Reply by tahaklkt-osxinfo** [reply]
@chris1111 what about big sur mate ? Im still waiting for big sur patch :'(
